code/game/objects/effects/landmarks/excavation_site.dm

```diff
@@ -1,5 +1,7 @@
 /obj/effect/landmark/excavation_site
 	name = "excavation landmark"
+	///Spawner the excavation site was spawned by
+	var/obj/effect/landmark/excavation_site_spawner/spawner
```

Comment:
yeah you're making this too complex that it should.

Reply:
ditto